src/xnatdcat/xnat_parser.py

```diff
@@ -1,17 +1,18 @@
 """Simple tool to query an XNAT instance and serialize projects as datasets"""
+import logging
+
 from rdflib import DCAT, DCTERMS, FOAF, Graph, Namespace, URIRef
 from rdflib.term import Literal
 from tqdm import tqdm
-import logging
 
-from .dcat_model import DCATDataSet, VCard
+from .dcat_model import DCATCatalog, DCATDataSet, VCard
 
 VCARD = Namespace("http://www.w3.org/2006/vcard/ns#")
 
 logger = logging.getLogger(__name__)
 
 
-def xnat_to_DCATDataset(project) -> DCATDataSet:
+def xnat_to_DCATDataset(project, config) -> DCATDataSet:
```

Comment:
Could add type hints for the arguments.

Reply:
Type hints are added for all functions in this file.